libexec/rc/rc.conf

```diff
@@ -672,6 +672,9 @@ kern_securelevel="-1"	# range: -1..3 ; `-1' is the most insecure
 			# in the system booting with securelevel set to 1, as
 			# init(8) will raise the level when rc(8) completes.
 update_motd="YES"	# update version info in /etc/motd (or NO)
+motd_perf_warnings_top="NO"	# Add warnings about performace reducing options
+				# to the top of motd
+motd_perf_warnings_bottom="YES"	# As above, but at the bottom
```

Comment:
I feel like having both supported is a bit overkill, but I guess it's simple to add...